src/spreadinterp.cpp

```diff
@@ -2185,26 +2170,6 @@ template FINUFFT_EXPORT_TEST int setup_spreader<double>(
     finufft_spread_opts &opts, double eps, double upsampfac, int kerevalmeth, int debug,
     int showwarn, int dim, int spreadinterponly);
 
-template<typename T>
-T evaluate_kernel(T x, const finufft_spread_opts &opts)
-/* ES ("exp sqrt") kernel evaluation at single real argument:
-      phi(x) = exp(beta.(sqrt(1 - (2x/n_s)^2) - 1)),    for |x| < nspread/2
-   related to an asymptotic approximation to the Kaiser--Bessel, itself an
-   approximation to prolate spheroidal wavefunction (PSWF) of order 0.
-   This is the "reference implementation", used by eg finufft/onedim_* 2/17/17.
-   Rescaled so max is 1, Barnett 7/21/24
-*/
-{
-  if (abs(x) >= (T)opts.ES_halfwidth)
-    // if spreading/FT careful, shouldn't need this if, but causes no speed hit
-    return 0.0;
-  else
-    return exp((T)opts.ES_beta * (sqrt((T)1.0 - (T)opts.ES_c * x * x) - (T)1.0));
-}
-
-template float evaluate_kernel<float>(float x, const finufft_spread_opts &opts);
-template double evaluate_kernel<double>(double x, const finufft_spread_opts &opts);
-
 template<typename T, uint8_t ns, uint8_t upsampfact>
 T evaluate_kernel_horner_kernel(T x, const finufft_spread_opts &opts)
```

Comment:
remind me - the decision was to leave this as is (& using the matlab-generated .h files), for now, right? Then kill them in the next PR?